dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -46,15 +46,15 @@ MENU_003 MENU
 BEGIN
     MENUITEM "&Retroceder\tCtrl+Z", FCIDM_SHVIEW_UNDO
     MENUITEM SEPARATOR
-    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT
+    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT //FIXME: accelerator collision &T
     MENUITEM "&Copiar\tCtrl+C", FCIDM_SHVIEW_COPY
     MENUITEM "Co&lar\tCtrl+V", FCIDM_SHVIEW_INSERT
     MENUITEM "Colar &atalho", FCIDM_SHVIEW_INSERTLINK
     MENUITEM SEPARATOR
     MENUITEM "Copiar &pasta...", FCIDM_SHVIEW_COPYTO
     MENUITEM "Mo&ver para pasta...", FCIDM_SHVIEW_MOVETO
     MENUITEM SEPARATOR
-    MENUITEM "Seleccionar &tudo\tCtrl+A", FCIDM_SHVIEW_SELECTALL
+    MENUITEM "Seleccionar &tudo\tCtrl+A", FCIDM_SHVIEW_SELECTALL //FIXME: accelerator collision &T
```

Comment:
&Seleccionar seems to me a good option on line 57

Reply:
Sounds good to me, but please open a follow-up-PR, I am almost done with the backmerge now (already at 0.4.7) and I don't want to reiterate again from the start.